capstone/src/main/java/com/google/sps/data/Form.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+import java.io.*;
+import com.google.appengine.api.datastore.Entity;
+
+
+public class Form {
+
+    private String editURL;
+    private String URL;
+    private int id;
+    private boolean isPublished;
+
+    public Form(String editURL, String URL, int id) {
+        this.editURL = editURL;
+        this.URL = URL;
+        this.id = id;
+        this.isPublished = false;
+    }
+
+    // Setters
```

Comment:
Comments like this are unecessary.

Adding a comment to the top of each class giving a general overview of the class as a whole is much more useful